src/components/Transaction.tsx

```diff
@@ -44,6 +44,14 @@ export const Transaction: React.FunctionComponent<TransactionProps> = props => {
       >
         {props.el.elab_ranking ? t("Yes") : t("No")}
       </div>
+      <div
+        className={classNames({
+          "col-sm-1": true,
+          revert: props.el.operation_type === "01"
```

Comment:
may you add a comment on what "01" means in this context? a better approach would have been using some kind of enumeration to make the intent clearer

Reply:
the `fn-backoffice` use this function for param conversion 
```
export function getOperationType(operationType: string): OperationType {
  switch (operationType) {
    case "00":
      return OperationType.Payment;
    case "01":
      return OperationType.Transfer;
    case "02":
      return OperationType.ApplePay;
    case "03":
      return OperationType.GooglePay;
    default:
      return OperationType.Other;
  }
}
```
We can use something like that or otherwise we can check the value of `operation_type_descr`
```suggestion
          revert: props.el.operation_type_descr === OperationType.Transfer
```

Reply:
Yup, this suffices: 

```
enum OperationType = { Transfer = "01", ... }
...
props.el.operation_type_descr === OperationType.Transfer
```

Reply:
`OperationType` already exists from the api specs as `Operation_type_descrEnum` to map `operation_type_descr` values.
The new enum you propose map the values of `operation_type` instead.